### PR TITLE
Don't apply indentation inference for `:style/indent nil` metadata

### PR DESCRIPTION
**Pull request by vemv**
> Fixes https://github.com/clojure-emacs/cider/issues/3684

### CODE REVIEW

File: src/cider/nrepl/middleware/track_state.clj [hunk 1]
```diff
@@ -31,8 +31,8 @@
   [m]
   (and (:macro m)
        (:arglists m)
-       (not (:style/indent m))
-       (not (:indent m))
+       (not (contains? m :style/indent))
+       (not (contains? m :indent))
```
Comment (alexander-yakushev) [inline src/cider/nrepl/middleware/track_state.clj:35]: Is this a backward-compability wart? I've never seen anybody use `:indent`.

**Review by alexander-yakushev — COMMENTED** [review]
Looks good!

File: src/cider/nrepl/middleware/track_state.clj (hunk 1 above)
Comment (vemv) [reply src/cider/nrepl/middleware/track_state.clj:35]: Me neither, but from memory, it's always been there internally

**Comment by vemv** [comment]
Thanks for the eyeing!

**Pull request merged**